account_statement_import_base/models/account_journal.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2022 Akretion France (http://www.akretion.com/)
+# @author: Alexis de Lattre <alexis.delattre@akretion.com>
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl).
+
+from odoo import api, models
+
+from odoo.addons.base.models.res_bank import sanitize_account_number
+
+
+class AccountJournal(models.Model):
+    _inherit = "account.journal"
+
+    def _statement_line_import_speeddict(self):
```

Comment:
`speeddict`?

Reply:
speeddict is a dict to improve speed. It allows you to pre-fetch some data, which may avoid to execute some search() requests for every statement line to find the relevant data (for partner matching for example). You can find an example of the use of speeddict in my module account_statement_completion_label_simple, cf this code: https://github.com/akretion/bank-statement-reconcile-simple/blob/14-compat-import_base/account_statement_completion_label_simple/models/account_journal.py#L97

Reply:
Gotcha, still `speeddict` as a name feels odd if it requires that explanation 🤔

Reply:
IDK, even `dict` would've been odd there. Sounds like `_prefill_statement_line_import_data` to be..